src/sentry/static/sentry/app/components/sampleTransitioner/sampleContainer.jsx

```diff
@@ -2,28 +2,13 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import SampleWell from './sampleWell';
 import { SampleLocation } from './sampleLocation';
+import { Sample } from './sample';
 
-export const SampleContainerType = {
+export const SampleContainerDirectionality = {
```

Comment:
I wonder if `SampleContainerSide` could be simpler to type and read? As in "it's on the source side (left), not the target side (right)".